demisto_sdk/tests/integration_tests/secrets_integration_test.py

```diff
@@ -67,3 +69,39 @@ def test_integration_secrets_negative(mocker):
     assert "Dynamics365ForMarketingEmail" in result.stdout
     assert "Remove or whitelist secrets in order to proceed, then re-commit" in result.stdout
     assert result.stderr == ""
+
+
+def test_integration_secrets_integration_positive(mocker):
+    """
+    Given
+    - FeedAzure integration yml with secrets.
```

Comment:
in the test description you don't mention the whitelist file. And what exactly you are whitelisting and which secrets your file contains.
Also your test do not address the pack .secrets_ignore file

Reply:
I've just copied an already existing test and modified it. 
.secrets_ignore already tested in other tests.

Reply:
From the description of the test reader can't understand what is the secret that the yml contains, and you don't mention that you have whitelist file that whitelists that secret.